lib/navigation.js

```diff
@@ -30,23 +30,35 @@ module.exports = function () {
             let frontmatter = files[url + '/index.html']
             // if we have frontmatter for that file, create subitem
             if (frontmatter) {
-              if (frontmatter.status && frontmatter.status === 'Draft') {
-                // If the page is labelled as a draft, do not show in the navigation.
-                continue
-              }
-
               let subitem = {
                 'url': url,
                 'label': frontmatter.title
               }
               // if frontmatter contains `theme` data, attach it to navigation item for grouping
-              if (frontmatter.theme) {
+              if (frontmatter.theme !== undefined) {
```

Comment:
It's not clear why we need to change these checks.

If we do want to do this we should use `typeof frontmatter.theme !== "undefined"`.